### PR TITLE
Issue226: Add in Alarm Alert Functionality to home.html

### LINKED ISSUE #226: Add in Alarm Alert Functionality to home.html

### PR DESCRIPTION
**Pull request by Reebs296**
Fixes #226

### CODE REVIEW

**Comment by Reebs296** [comment]
I will fix formatting in a separate issue.

**Review by JamesShadowfyre — APPROVED** [review]
Side bar looks good on various screen sizes. As long as this matches what @brkykrclk is currently implementing for alarm functionality, this is approved.

**Comment by Jcp149** [comment]
im not in love with the new design with a side bar for the alarms, and the alarm text for max and min is not formatted perfect there is an "alarm" right next to the input box

**Comment by Reebs296** [comment]
@Jcp149 can you show me what you see? I didnt have those issues. This was the layout that Berk requested when we spoke during the previous lab session. Right now we don't even have alarms working in the backend, so it's just a format template.

**Comment by Jcp149** [comment]
Screenshot 2024-04-01 at 9 49 43 PM

File: src/flaskr/templates/main/home.html [hunk 1]
```diff
@@ -4,6 +4,7 @@
   {% block title %}Home Page | Operations Optimization{% endblock %}
 {% endblock %}
 
+
 {% block content %}
   <script src="https://cdn.jsdelivr.net/npm/chart.js@2.9.4/dist/Chart.min.js"></script>
```
Comment (JamesShadowfyre) [inline src/flaskr/templates/main/home.html:10]: These lines below should be added to the indented block

**Review by JamesShadowfyre — CHANGES_REQUESTED** [review]
Lines 10-18 in the Home.html should be added to the form block content. Currently they are cut off from the side bar and half covered.

**Review by Birdbh — APPROVED** [review]

**Review by Birdbh — DISMISSED** [review]
image
The Alarm sidebar cuts off the chart when displayed.

**Comment by Reebs296** [comment]
I have fixed the formatting issue with the charts, please see now.

**Review by Jcp149 — APPROVED** [review]
Looks great. thanks for fixing it all and making it look pretty

**Pull request merged**